lib/csetgrp.gi

```diff
@@ -80,6 +80,29 @@ local c,i;
   fi;
 end );
 
+
```

Comment:
Typo in one commit message: "permuattion"

Reply:
changed.